aeron-cluster/src/test/java/io/aeron/cluster/TestCluster.java

```diff
@@ -54,7 +59,7 @@
     private static final String ARCHIVE_CONTROL_RESPONSE_CHANNEL =
         "aeron:udp?term-length=64k|endpoint=localhost:8020";
     private static final String ARCHIVE_RECORDING_EVENTS_CHANNEL =
-        "aeron:udp?control-mode=dynamic|control=localhost:8030";
+        "aeron:udp?endpoint=235.0.0.1:8030";
```

Comment:
Not good to use multicast in tests as issues can occur on Windows if publication is setup before subscriptions. I've disabled recording events for cluster tests.